tests/console/shells.pm

```diff
@@ -32,7 +33,7 @@ sub run() {
     select_console 'root-console';
     my @packages = qw(tcsh zsh);
     # ksh does not build for Leap 15.x on aarch64, so, skip it
-    push @packages, qw(ksh) unless (is_leap('15.0+') and check_var('ARCH', 'aarch64'));
+    tcsh zshpush @packages, qw(ksh) unless (is_leap('15.0+') and check_var('ARCH', 'aarch64'));
```

Comment:
Maybe this could be in a separate PR.

Reply:
This looks like an "oops", I don't know where this came from.